packages/core/src/queries/application.ts

```diff
@@ -49,11 +48,6 @@ export const deleteApplicationById = async (id: string) => {
     where id=${id}
   `);
   if (rowCount < 1) {
-    throw new RequestError({
-      code: 'entity.not_exists_with_id',
-      name: Applications.tableSingular,
-      id,
-      status: 404,
-    });
+    throw new NotFoundError();
```

Comment:
per discussion this should be a 500? also do we need wrap the logic into an util? cc @wangsijie

Reply:
Throw a general error? A bit different from other not found cases.